runtime/build.gradle.kts

```diff
@@ -78,16 +88,12 @@ kotlin {
 
 tasks {
     val generateWellKnownTypes by registering(ProtocTask::class) {
-        val protocPath = provider {
-            System.getProperty("protoc.path")
-                ?: throw InvalidUserDataException("System property protoc.path must be set")
-        }.map { rootProject.layout.projectDirectory.dir(it) }
-
-        includeDir.set(protocPath.map { it.dir("include") })
+        includeDir.set(project.file("$protoDir/src"))
         outputDir.set(project.file("src/commonMain/kotlin"))
         kotlinPackage.set("pbandk.wkt")
         logLevel.set("debug")
         protoFileSubdir("google/protobuf")
+        dependsOn("gitQueryTask")
```

Comment:
This allows us to avoid manually syncing the proto repo for the includes. It also guarantees that we stay pinned to particular version (hopefully the same one as the global project protobuf version)

Reply:
Do you mind pulling this out into a separate PR? I like the idea but I have a couple questions about the `gitquery` plugin, and if we're going to use this plugin I'd like to use it for the .proto files in the `conformance` and `protoc-gen-kotlin` subprojects as well.

Reply:
done